roo_on_rails.gemspec

```diff
@@ -30,7 +30,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency 'dotenv-rails', '~> 2.1'
+  spec.add_runtime_dependency 'newrelic_rpm', '~> 3.17'
+  spec.add_runtime_dependency 'rails', '~> 5.0'
+
   spec.add_development_dependency "bundler", "~> 1.13"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency 'thor', '~> 0.19'
+  spec.add_development_dependency 'pry'
```

Comment:
I hate to be that guys... but why not `pry-byebug` then?
Also because using `pry` instead of `irb` for `rails console` is a must.